boards/firechip/base-workloads/ubuntu-base/README

```diff
@@ -0,0 +1,5 @@
+Ubuntu 20.04 support in Firesim
```

Comment:
Can you move this to a docs page in https://github.com/firesim/FireMarshal/tree/master/docs/source/distros

Reply:
Done, added one there